pkg/plugins/golang/v3/init.go

```diff
@@ -207,6 +207,9 @@ func checkDir() error {
 			if err != nil {
 				return err
 			}
+			if info.IsDir() && info.Name() != "." && strings.HasPrefix(info.Name(), ".") {
+				return filepath.SkipDir
+			}
```

Comment:
Will not it allow scaffold projects in a directory that has another directory as well? 
Then, note that it goes against the purpose of the check. We would like to not allow it because it means that we cannot ensure that the tool is able to do properly all required scaffolds.

Reply:
I believe the check is too paranoid. it needs to not only exclude files starting with a dot, it needs to not recurse into hidden directories.

It will only allow you to scaffold into directories that have hidden directories at the root. For example, if you try to scaffold right now in a directory that you've just run a `git init` on, it'll fail. Though `.git` has a `.` prefix, the `HEAD` file in `.git/HEAD` does not, so it'll fail because `info.Name()' will return `HEAD`, not `.git/HEAD`.

Similarly if you have opened that directory in an editor already and something like a `.vscode` exists it'll fail to scaffold. None of these directories should have any impact on the ability to scaffold. If we ignore hidden files, we should ignore hidden directories too.